doc/source/whatsnew/v0.24.0.txt

```diff
@@ -1344,6 +1344,7 @@ Reshaping
 - Bug in :func:`pandas.concat` when concatenating a multicolumn DataFrame with tz-aware data against a DataFrame with a different number of columns (:issue`22796`)
 - Bug in :func:`merge_asof` where confusing error message raised when attempting to merge with missing values (:issue:`23189`)
 - Bug in :meth:`DataFrame.nsmallest` and :meth:`DataFrame.nlargest` for dataframes that have :class:`MultiIndex`ed columns (:issue:`23033`).
+- Bug in :func: `pandas.melt` when passing column names that do no exist in dataframe (:issue:`23575`)
```

Comment:
Say a bit about what the bug was (i.e. not raise an error).

"that do no exist in dataframe" -> "not present in the \`\`DataFrame\`\`"